src/tests/multihost/admultidomain/test_multidomain.py

```diff
@@ -153,3 +154,76 @@ def test_0002_bz2018432(multihost, newhostname, adjoin):
+
+    @staticmethod
+    def test_0003_bz1913284_keytab_as_nonroot(multihost, newhostname, adchildjoin):
+        """
+        :title: IDM-SSSD-TC: ad_provider: forests: krb5_kt_start_seq_get failed:
+          Permission denied when running as unprivileged user sssd
+        :id: 53a6871e-95a6-4865-9e61-1e12815ec35b
+        :setup:
+          1. Configure parent and child domain
+          2. Join client to child domain
+        :steps:
+          1. Lookup user from child domain
+          2. Lookup user from parent domain
+          3. Check log for a keytab error
+        :expectedresults:
+          1. Parent user is found
+          2. Child user is found
+          3. The permission denied error is not present in log.
+        :customerscenario: True
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1913284
+        """
+        adchildjoin(membersw='adcli')
+        ad_domain = multihost.ad[0].domainname
+        child_domain = multihost.ad[1].domainname
+        ad_server = multihost.ad[1].hostname
+
+        # Configure sssd
+        multihost.client[0].service_sssd('stop')
+        client = sssdTools(multihost.client[0], multihost.ad[1])
+        client.backup_sssd_conf()
+        dom_section = f'domain/{client.get_domain_section_name()}'
+        sssd_params = {
+            'ad_domain': child_domain,
+            'debug_level': '9',
+            'use_fully_qualified_names': 'True',
+            'ad_server': ad_server,
+            'cache_credentials': 'True',
+        }
+        client.sssd_conf(dom_section, sssd_params)
+        # Set sssd to run as user sssd
+        with tempfile.NamedTemporaryFile(mode='w') as tfile:
+            tfile.write("[sssd]\nuser = sssd\n")
+            tfile.flush()
+            multihost.client[0].transport.put_file(tfile.name, '/etc/sssd/conf.d/nonroot-test.conf')
+        client.clear_sssd_cache()
+
+        # Search for the user in root domain
+        getent_root_user1 = multihost.client[0].run_command(
+            f'getent passwd user1@{ad_domain}',
+            raiseonerr=False
+        )
+        # Search for the user in child domain
+        getent_child_user1 = multihost.client[0].run_command(
+            f'getent passwd child_user1@{child_domain}',
+            raiseonerr=False
+        )
+        # Download sssd log
+        log_str = multihost.client[0].get_file_contents(
+            f"/var/log/sssd/sssd_{multihost.ad[0].domainname.lower()}.log"). \
+            decode('utf-8')
+        multihost.client[0].run_command(
+            'rm -f /etc/sssd/conf.d/nonroot-test.conf',
+            raiseonerr=False
+        )
+        client.restore_sssd_conf()
+        client.clear_sssd_cache()
+
+        # Evaluate test results
+        assert getent_root_user1.returncode == 0
+        assert getent_child_user1.returncode == 0
+        assert "krb5_kt_start_seq_get failed: Permission denied" not in log_str
+        assert "Failed to read keytab [FILE:/etc/krb5.keytab]: No " \
```

Comment:
maybe would be better to use a wildcard instead of hardcoding path to keytab (`/etc/krb5.keytab`)

Reply:
Why? This is a test about permissions/access to keytab.

Reply:
Path can be configured using `krb5_keytab` in sssd.conf

Reply:
I do not have it configured explicitly so default name/path make most sense.